File: redsheet-portal/src/app/database-models/v-main-module-master.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vMainModuleMaster {
    constructor(vMainModuleMaster?: vMainModuleMaster )  {
        let properties = [ "isRolePermissionItem", "moduleMasterId", "moduleMasterName",];
        for (let property of properties)
            if (vMainModuleMaster && vMainModuleMaster[property])
                this[property] = vMainModuleMaster[property];
    }
 
	isRolePermissionItem : boolean = false ;
 
	moduleMasterId : number =   0 ;
 
    @required()
    @maxLength(100)
	moduleMasterName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vMainModuleMaster.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vMainModuleMasters", Schema = "dbo")]
	[Serializable]
	public partial class vMainModuleMaster  
	{

	public Nullable<bool> IsRolePermissionItem { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int ModuleMasterId { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string ModuleMasterName { get; set; } 

	public vMainModuleMaster()
	{
    }

    }
}

